src/util/constants.py

```diff
@@ -1,4 +1,27 @@
+from enum import Enum, EnumMeta
+
 SENTENCES_PATH = './static/sentences'
 
 LABEL_IDS = ['NOT_RELEVANT', 'CAUSE_1', 'CAUSE_2', 'CAUSE_3', 'EFFECT_1', 'EFFECT_2', 'EFFECT_3', 'AND', 'OR', 'VARIABLE', 'CONDITION', 'NEGATION']
-LABEL_IDS_VERBOSE = ['notrelevant', 'Cause1', 'Cause2', 'Cause3', 'Effect1', 'Effect2', 'Effect3', 'Conjunction', 'Disjunction', 'Variable', 'Condition', 'Negation']
\ No newline at end of file
+LABEL_IDS_VERBOSE = ['notrelevant', 'Cause1', 'Cause2', 'Cause3', 'Effect1', 'Effect2', 'Effect3', 'Conjunction', 'Disjunction', 'Variable', 'Condition', 'Negation']
```

Comment:
Could the items of `LABEL_IDS_VERBOSE`, which have been extracted into independent variables (like `CONJUNCTION` or `NOTRELEVANT`), replace the hard-coded values in the list without breaking the existing code? Otherwise, we have redundancy in this file, as one could change one string (e.g., "notrelevant") but leave the other instance as is.

Reply:
Yes, I would like to do this in a separate PR, after the other parts of the code base are refactored to use the constants.